src/plugins/gui/providers/moonpayProvider.ts

```diff
@@ -163,7 +164,7 @@ const MOONPAY_PAYMENT_TYPE_MAP: Partial<Record<FiatPaymentType, MoonpayPaymentMe
   applepay: 'credit_debit_card',
   credit: 'credit_debit_card',
   googlepay: 'credit_debit_card',
-  iach: 'ach_bank_transfer',
```

Comment:
What is `iach` and why is it replaced by `ach`?

Reply:
iach is a purchase only ach method which we were using with moonpay years ago. now we're only enabling an `ach` sell method. moonpay uses the same code but iach isn't supported for sell, only buy.